ingestion_server/test/integration_test.py

```diff
@@ -267,6 +271,43 @@ def _promote(self, model, suffix="integration", alias=None):
         es = self._get_es()
         assert list(es.indices.get(index=alias).keys())[0] == f"{model}-{suffix}"
 
+    def _delete_index(self, model, suffix="integration", alias=None):
+        req = {
+            "model": model,
+            "action": "DELETE_INDEX",
+            "callback_url": bottle_url,
+        }
+        if alias is None:
+            req |= {"index_suffix": suffix}
+        else:
+            req |= {
+                "alias": alias,
+                "force_delete": True,
+            }
+        res = requests.post(f"{ingestion_server}/task", json=req)
+        stat_msg = "The job should launch successfully and return 202 ACCEPTED."
+        self.assertEqual(res.status_code, 202, msg=stat_msg)
+
+        # Wait for the task to send us a callback.
+        assert self.__class__.cb_queue.get(timeout=120) == "CALLBACK!"
+
+        es = self._get_es()
+        with pytest.raises(NotFoundError):
+            es.indices.get(index=f"{model}-{suffix}")
+
+    def _soft_delete_index(self, model, alias, suffix="integration", ambiguous=False):
+        req = {
+            "model": model,
+            "action": "DELETE_INDEX",
+            "alias": alias,
+        }
+        if not ambiguous:
+            req |= {"force_delete": False}
```

Comment:
What does `ambiguous` mean in this context? Could the parameter be named `omit_force_delete` instead? Maybe a doc comment would help explain this. It took me a couple read throughs to understand what this test utility does.